pom.xml

```diff
@@ -57,8 +57,11 @@
             <configuration>
               <rules>
                 <requireMavenVersion>
-                  <version>[3.5.4,)</version>
+                  <version>[3.8.6,)</version>
                 </requireMavenVersion>
+                <requireJavaVersion>
+                  <version>[11,)</version>
```

Comment:
> Will need to confirm that it still works with Java 8 and Java 11.

2.375.1 does not support compiling with Java 8 anymore, but if we want to keep Java 8 for some reason, we need to lower this boundary.

Reply:
Thanks.  You're right.  We only need to confirm that it works with Java 11.  I see no issue in the change for either Java 11 or Java 17.